staging/src/kubevirt.io/client-go/apis/core/v1/types.go

```diff
@@ -1840,6 +1849,14 @@ type StartOptions struct {
 	// Indicates that VM will be started in paused state.
 	// +optional
 	Paused bool `json:"paused,omitempty" protobuf:"varint,7,opt,name=paused"`
+	// When present, indicates that modifications should not be
+	// persisted. An invalid or unrecognized dryRun directive will
+	// result in an error response and no further processing of the
+	// request. Valid values are:
+	// - All: all dry run stages will be processed
+	// +optional
+	// +listType=atomic
+	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,5,rep,name=dryRun"`
```

Comment:
Could you please explain the protobuf fields here, and why the second value is 5 on this one and 2 on the others?
(Feel free to just point me to the documentation, I just couldn't find it!)

Reply:
/hold
Feel free to unhold, it just makes sense to keep this open while there's an open question.

Reply:
I think it was just a mistake. Here's the [documentation](https://developers.google.com/protocol-buffers/docs/overview#assigning_field_numbers). If I understand correctly the second field basically is a unique identifier "_used to identify your fields in the message binary format_".

Reply:
/unhold